runtime/doc/nvim-extra-keymaps.txt

```diff
@@ -0,0 +1,35 @@
+*nvim-extra-keymaps.txt*    Extra default keymaps                        {Nvim}
+For nvim.    Last change:      2015 May
+
+*nvim-extra-keymaps*
+
+The `plugin/extra-keymaps.vim` script provides a few extra keymaps that might
+be useful and tries to fix some issues the community has identified with vim's
+default keymaps.
+
+The user can disable the plugin by having >
+
+    let g:nvim#defaults#maps#use = 0
+
+in their |vimrc| (additionally, all features can be disabled separately,
+using the variables described below).
+
+=============================================================================
+The keymaps provided are:
+
+                                        *gr:nvim#defaults#maps#leave_term_mode*
+1. Leave |terminal-mode| using <esc><esc>, besides the usual  CTRL-\ CTRL-N
+   (see |c_CTRL_\_CTRL_N|).
+
+                                                  *g:nvim#defaults#maps#Y_eol*
+2. |Y| yanks to the end of line.
+
+                                            *g:nvim#defaults#maps#undo_ctrl_u*
+3. <C-u> can be undoed. (see |i_CTRL-U|)
+
+                                             *g:nvim#defaults#maps#smart_home*
+4. <Home> goes to the beginning of the text on first press and the beginning
```

Comment:
It means the first non-space character in the current line. I took it from the initial description of the mapping, I can see how it's ambiguous. :+1: